src/Wpf.Ui/Resources/Theme/HC2.xaml

```diff
@@ -349,7 +349,13 @@
     <SolidColorBrush x:Key="HyperlinkButtonForegroundDisabled" Color="{StaticResource SystemColorGrayTextColor}" />
 
     <!--  InfoBadge  -->
-    <!--  TODO  -->
+    <SolidColorBrush x:Key="InfoBadgeValueForeground" Color="{StaticResource TextFillColorInverse}" />
```

Comment:
We can't be using `TextFillColorInverse` in HC themes. You'd problaby want to use `SystemColorButtonTextColor` instead

```suggestion
    <SolidColorBrush x:Key="InfoBadgeValueForeground" Color="{StaticResource SystemColorButtonTextColor}" />
```